gax/test/showcase-echo-client/src/v1beta1/echo_client.ts

```diff
@@ -18,23 +18,11 @@
 
 /* global window */
 import * as gax from 'google-gax';
-import type {
-  Callback,
-  CallOptions,
-  Descriptors,
-  ClientOptions,
-  GrpcClientOptions,
-  LROperation,
-  PaginationCallback,
-  GaxCall,
-  IamClient,
-  IamProtos,
-  LocationsClient,
-  LocationProtos,
-} from 'google-gax';
+import type {Callback, CallOptions, Descriptors, ClientOptions, GrpcClientOptions, LROperation, PaginationCallback, GaxCall, IamClient, IamProtos, LocationsClient, LocationProtos} from 'google-gax';
```

Comment:
would this change be fixed by a lil `npm run fix`?

Reply:
So this library doesn't have gts, I did run npm fix at the top level but it doesn't touch these files. I think in a future PR maybe I'll add something for linting in the generator, see: https://github.com/googleapis/gapic-generator-typescript/issues/1698